doc/lsst.pipe.tasks/deblending-flags-overview.rst

```diff
@@ -0,0 +1,153 @@
+
+Datasets           isPatchInner  isTractInner  isIsolated  fromBlend  isDeblendedSource  isDeblendedModelSource  isPrimary
+---------------------------  ------------  ------------  ----------  ---------  -----------------  ----------------------  ---------
+      isolated sources            x             x            x                          x                                      x
+parents with multiple peaks       x             x
+ models of isolated sources       x             x            x                                               x
+models of deblended sources       x             x                        x              x                    x                 x
+        sky objects               x             x            x           x              x                    x
+   patch overlap regions                        x            x           x              x                    x
+   tract overlap regions          x                          x           x              x                    x
+---------------------------  ------------  ------------  ----------  ---------  -----------------  ----------------------  ---------
+
+Deblender input
+===============
+
+The input `mergeDet` catalog to the deblender contains a list of parent sources, each row consisting of information about the parent, a footprint (a boolean mask of pixels in the input image that were detected as part of the parent blend), and a peak catalog (a list of peak locations and central flux values for all detected peaks in the parent blend). From here things change depending on which deblender is used, as single visit deblending uses the single-band :py:module:`lsst.meas.deblender` while multi-band deblending uses :py:module:`lsst.meas.extensions.scarlet` and generates a slightly different set of outputs.
+
+Single-band deblender
+=====================
+
+Prior to the adoption of scarlet as the default deblender for coadds, all deblending was done with :py:class:`lsst.meas.deblender.SourceDeblendTask`, which is still the single visit (single band) deblender. The basic idea of the algorithm is that most galaxies roughly exhibit 180 degree symmetry and (at shallow depths) are only blended with one other object in most cases. So a symmetric template (numerically equivalent to `x = np.min([x, x[::-1]])`) is made for all of the sources in a blend that do not fit the PSF, and the flux in the image is re-apportioned to each source based on the ratio of the templates for each pixel. This algorithm can fail for non-symmetric sources and for any blends where a source is blended with neighbors on both sides, causing inaccuracies in the measurement of all three sources. As the depth of the images increases blending becomes more severe and the instances where this algorithm fails increases, which is why a different deblender is used for co-added images in multiple bands.
```

Comment:
```suggestion
Prior to the adoption of scarlet as the default deblender for coadds, all deblending was done with `lsst.meas.deblender.SourceDeblendTask`, which is still the single visit (single band) deblender. The basic idea of the algorithm is that most galaxies roughly exhibit 180 degree symmetry and (at shallow depths) are only blended with one other object in most cases. So a symmetric template (numerically equivalent to ``x = np.min([x, x[::-1]])``) is made for all of the sources in a blend that do not fit the PSF, and the flux in the image is re-apportioned to each source based on the ratio of the templates for each pixel. This algorithm can fail for non-symmetric sources and for any blends where a source is blended with neighbors on both sides, causing inaccuracies in the measurement of all three sources. As the depth of the images increases blending becomes more severe and the instances where this algorithm fails increases, which is why a different deblender is used for co-added images in multiple bands.
```

FYI, code literals use double-back ticks, which links to Python APIs use single backticks.